tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py3{7,8,9}
+envlist = py3{7,8,9}-pynetdicom{1,2}
```

Comment:
Good thinking!

Reply:
I ended up rolling this back since there were breaking changes in the interface between 1 and 2. I'm not sure 1 is very widely used; however, if it becomes a problem, it shouldn't be too difficult to add back in support and testing for pydicom <2 later.